BinaryObjectScanner/Protection/Roxxe.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+#if NET40_OR_GREATER || NETCOREAPP
+using System.Collections.Concurrent;
+#endif
+using System.Collections.Generic;
+using System.Linq;
+using BinaryObjectScanner.Interfaces;
+using SabreTools.Matching;
+using SabreTools.Serialization.Wrappers;
+
+namespace BinaryObjectScanner.Protection
+{
+    /// <summary>
+    /// Roxxe was a Czech DRM. It appears to have been a simple disc check that also relied on unusual disc manufacturing and dummy files to attempt to prevent copying.
+    /// 
+    /// DRML: https://github.com/TheRogueArchivist/DRML/blob/main/entries/Roxxe/Roxxe.md
+    /// </summary>
+    public class Roxxe : IPathCheck, IPortableExecutableCheck
+    {
+        /// <inheritdoc/>
+        public string? CheckPortableExecutable(string file, PortableExecutable pex, bool includeDebug)
+        {
+            // Get the sections from the executable, if possible
+            var sections = pex.Model.SectionTable;
+            if (sections == null)
+                return null;
+
+            // Get the CODE section strings, if they exist
+            var strs = pex.GetFirstSectionStrings("CODE");
+            if (strs != null)
+            {
+                // Found in "Owar.exe" in IA item "game4u-22-cd".
+                if (strs.Any(s => s.Contains("TRCHANGER.INI")))
+                    return "Roxxe";
+            }
+
+            // Get the .rsrc section strings, if they exist
+            strs = pex.GetFirstSectionStrings(".rsrc");
```

Comment:
Did you find that these live in any specific resource? If you can lock it down to a resource and not just the resource section, for example, a string table resource, please do that.

Reply:
Despite my best efforts, I have not been able to narrow this down yet. It also wasn't in application-defined resources despite that looking promising.

Reply:
My best efforts weren't good enough